cache/not_nydus.go

```diff
@@ -0,0 +1,25 @@
+//go:build !nydus
+// +build !nydus
+
```

Comment:
File name should be something like `compression.go` and `compression_nydus.go` rather than `not_nydus.go`.

Reply:
Thanks for the reply! will fix it.

Reply:
Fixed.